index.html

```diff
@@ -4,12 +4,219 @@
+              <li><a href="#">README</a></li>
+              <li><a href="#">GROTH QUARTERS</a></li>
+          
+              <li><a class="white-bar-sec-text" href="#">SHIFT</a></li>
+              <li><a class="white-bar-sec-text" href="#">NEURAL</a></li>
+            </ul>
+
+            <ul class="nav-icons flex float-right">
+              <li><a href="#" class="search">
+                <img src="./images/computer-icons-pittsburg-button-search-box-button-ede7b4f4cdaaba07daadc3f70cc319f7.png" alt="">
+              </a></li>
+              <li><a href="#" class="menu-icon float-right">
+                <img src="./images/Hamburger_icon.svg.png" alt="">
+              </a></li>
+            </ul>
+          </div>
+
+          
+        </div>
+      </div>
+      <div class="yellow-bar flex">
+        <ul>
+          <li>Calling young digital talent in the Netherlands – Be recognized in the T500 →	</li>
+          
+        </ul>
+      </div>
+    </nav>
+
+    <main>
+      <div class="grid-one">
+        <div class="cover-image">
+            <a class="cover" href="#"><img src="images/tnw-logo.png" alt="logo" class="logo"></a>
+            <div class="header1">
+                <h1>Microsoft is adding AI-powered text predictions to Word</h1>
+                <p>by Thomas Macaulay </p>
+            </div>
+        </div>
+  
+        <div>
+          <div class="cover-image">
+            <div class="header1">
+              <h1>Microsoft is adding AI-powered text predictions to Word</h1>
+              <p>by Thomas Macaulay </p>
+            </div>     
+          </div>
+  
+          <div class="cover-image">
+            <div class="header1">
+              <h1>Microsoft is adding AI-powered text predictions to Word</h1>
+              <p>by Thomas Macaulay </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="funding">
+        <div class="funding-title">
+          <a href="#"><h2>Latest funding rounds on Index.co →</h2> </a>
+          <div class="funding-tags">
+            <a href="#"><h6>See more</h6></a>
+            <a href="#"><h6>Get updates by email</h6></a>
+            <a href="#"><h6>Add info</h6></a>
+          </div>
+        </div>
+        <div class="funding-list">
+
+          <div class="funding-element">
+            <img src="./images/image-funding.webp" alt="">
+            <div class="funding-h3">
+              <h3>Ringover</h3>
+            </div>
+            <div class="funding-money">
+              <h4 class="money-text">$12M</h4>
+              <h5 class="money-text" >SERIES A</h5>
+            </div>
+            <div class="funding-links">
+              <a href="#" class="block">Telecommunications</a>
+              <a href="#" class="block">Expedition Growth Capital</a>
+            </div>
+            <div class="funding-desc">
+              <p>Ringover raises $12 million to bring enterprise communications to the cloud on VentureBeat
+              </p>
```

Comment:
This and all the other text at the bottom of the article in this section should also be a link tag

Reply:
done